events/forms.py

```diff
@@ -0,0 +1,33 @@
+from django import forms
+from .class_models.event_models import Event
+from .class_models.participant_model import EventParticipant
+from users.models import User
+
+TIME_FORMAT = ['%I:%M %p %d-%b-%Y']
+
+
+class EventCreationForm(forms.ModelForm):
+
+    def __init__(self, *args, **kwargs):
+        self.user_id = kwargs.pop('user_id')
+        super(EventCreationForm, self).__init__(*args, **kwargs)
+
+    title = forms.CharField(initial="", required=True, max_length=50)
+    location = forms.CharField(initial="", required=False, max_length=50)
+    description = forms.CharField(initial="", required=False)
+    date_time_start = date_time_end = forms.DateTimeField(
+        input_formats=TIME_FORMAT,
+    )
+
+    class Meta:
+        model = Event
+        fields = '__all__'
+
```

Comment:
Maybe, instead of overriding the input types in the html file, you can use widgets in the Meta class like this:
```suggestion
        widgets = {
            'description': Textarea(attrs={'cols':50, 'rows': 3}),
            'date_time_start': DateTimePickerInput(),
            'date_time_end': DateTimePickerInput(),
        }
```
while DateTimePickerInput will be a class as folow:
```suggestion
class DateTimePickerInput(forms.DateTimeInput):
    input_type = 'datetime-local'
```
that override the input type of the DateTime field (which is 'text' type by default).
You can look at this link: https://docs.djangoproject.com/en/4.0/topics/forms/modelforms/#overriding-the-default-fields
That explain how to override the default fields 😄 
I think in this way it will be easier to use the form later 😄

Reply:
```suggestion
class DataTimePickerInput(forms.DateTimeInput):
    input_type = "datetime-local",
    format="%Y-%m-%dT%H:%M"
```
In addition to @YaelD  comment
the format will help with the update of the form later on 😄

Reply:
done